gef.py

```diff
@@ -10647,6 +10686,26 @@ def invoke(self, args, from_tty):
         gdb.execute("gef help")
         return
 
+    def add_context_pane(self, pane_name, pane_function):
+        """Add a new context pane to ContextCommand"""
+        context_obj = None  # type: ContextCommand
+        for cmd, class_name, class_obj in self.loaded_commands:
+            if isinstance(class_obj, ContextCommand):
+                context_obj = class_obj
+                break
+
+        if not context_obj:
+            return False
+
+        # assure users can toggle the new context
+        corrected_settings_name = pane_name.replace(" ", "_")
+        layout_settings = context_obj.get_setting("layout")
+        context_obj.update_setting("layout", layout_settings+f" {corrected_settings_name}")
```

Comment:
f-strings appeared in Python3.7 and some GDB versions we support still ship with Python3.6. So this is more portable: 
```suggestion
        context_obj.update_setting("layout", "{} {}".format(layout_settings, corrected_settings_name))
```